### PR TITLE
Post Carousel: Add link target to items with thumbnail

### PR DESCRIPTION
**Pull request by AlexGStapleton**
Resolve https://github.com/siteorigin/so-widgets-bundle/issues/1154

The second commit is more about tidying up as things aren't very readable without it.

### CODE REVIEW

**Comment by Misplon** [comment]
Super, thanks. 

We don't seem to be applying the link target to the post title.

**Review by Misplon — CHANGES_REQUESTED** [review]
Looking good, we just need to apply the link target setting to the link in the post title.

**Review by Misplon — APPROVED** [review]

File: widgets/post-carousel/tpl/carousel-post-loop.php [hunk 1]
```diff
@@ -6,18 +6,26 @@
 while($posts->have_posts()) : $posts->the_post(); ?>
 	<div class="sow-carousel-item" tabindex="-1">
 		<div class="sow-carousel-thumbnail">
-			<?php if( has_post_thumbnail() ) : $img = wp_get_attachment_image_src(get_post_thumbnail_id(), $instance['image_size']); ?>
-				<a href="<?php the_permalink() ?>" style="background-image: url(<?php echo sow_esc_url($img[0]) ?>)" aria-labelledby="sow-carousel-id-<?php echo the_ID(); ?>" tabindex="-1">
+			<?php if ( has_post_thumbnail() ) : $img = wp_get_attachment_image_src( get_post_thumbnail_id(), $instance['image_size'] ); ?>
+				<a href="<?php the_permalink() ?>"
+					<?php echo $link_target == 'new' ? 'target="_blank" rel="noopener noreferrer"' : '' ?>
```
Comment (gregpriday) [inline widgets/post-carousel/tpl/carousel-post-loop.php:11]: Do we want `noopener noreferrer` to be the default here? I can understand that we use this for links inside the WordPress admin, but is it our place to block a referrer in a user's frontend? They might want/need this information to be passed.

**Review by gregpriday — COMMENTED** [review]
Code all looks good, just a general question.

File: widgets/post-carousel/tpl/carousel-post-loop.php (hunk 1 above)
Comment (AlexGStapleton) [reply widgets/post-carousel/tpl/carousel-post-loop.php:11]: Replied over Slack

**Review by gregpriday — APPROVED** [review]

**Pull request merged**